[2025-01-12T13:00:15.895+0000] {processor.py:186} INFO - Started process (PID=72) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:15.901+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:00:15.914+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:00:15.912+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:15.982+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:00:15.965+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:00:15.985+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:16.106+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.234 seconds
[2025-01-12T13:00:46.554+0000] {processor.py:186} INFO - Started process (PID=137) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:46.555+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:00:46.558+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:00:46.558+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:46.578+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:00:46.574+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:00:46.580+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:46.606+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.059 seconds
[2025-01-12T13:01:17.313+0000] {processor.py:186} INFO - Started process (PID=202) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:17.315+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:01:17.317+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:01:17.317+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:17.340+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:01:17.336+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:01:17.345+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:17.370+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.065 seconds
[2025-01-12T13:01:47.431+0000] {processor.py:186} INFO - Started process (PID=267) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:47.432+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:01:47.434+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:01:47.434+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:47.446+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:01:47.444+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:01:47.449+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:47.465+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.039 seconds
[2025-01-12T13:02:17.855+0000] {processor.py:186} INFO - Started process (PID=332) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:17.857+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:02:17.859+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:02:17.859+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:17.873+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:02:17.870+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:02:17.874+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:17.902+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.052 seconds
[2025-01-12T13:02:48.353+0000] {processor.py:186} INFO - Started process (PID=397) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:48.362+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:02:48.364+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:02:48.364+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:48.376+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:02:48.374+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:02:48.378+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:48.393+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.044 seconds
[2025-01-12T13:03:18.439+0000] {processor.py:186} INFO - Started process (PID=462) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:18.440+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:03:18.442+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:03:18.442+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:18.456+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:03:18.453+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:03:18.458+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:18.485+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.050 seconds
[2025-01-12T13:03:48.670+0000] {processor.py:186} INFO - Started process (PID=527) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:48.672+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:03:48.675+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:03:48.675+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:48.695+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:03:48.692+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:03:48.697+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:48.726+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.062 seconds
[2025-01-12T13:04:19.186+0000] {processor.py:186} INFO - Started process (PID=592) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:19.187+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:04:19.189+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:04:19.189+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:19.202+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:04:19.199+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:04:19.203+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:19.228+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.047 seconds
[2025-01-12T13:04:35.345+0000] {processor.py:186} INFO - Started process (PID=651) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:35.346+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:04:35.347+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:04:35.347+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:35.379+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:04:35.376+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:04:35.381+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:35.400+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.060 seconds
[2025-01-12T13:05:05.764+0000] {processor.py:186} INFO - Started process (PID=716) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:05:05.765+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:05:05.767+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:05:05.767+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:05:05.783+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:05:05.779+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:05:05.784+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:05:05.800+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.041 seconds
[2025-01-12T13:09:10.307+0000] {processor.py:186} INFO - Started process (PID=66) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:09:10.309+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:09:10.313+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:09:10.313+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:09:10.377+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:09:10.371+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:09:10.379+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:09:10.419+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.125 seconds
[2025-01-12T13:09:40.739+0000] {processor.py:186} INFO - Started process (PID=132) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:09:40.740+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:09:40.742+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:09:40.742+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:09:40.757+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:09:40.753+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:09:40.759+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:09:40.788+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.055 seconds
[2025-01-12T13:10:11.765+0000] {processor.py:186} INFO - Started process (PID=197) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:10:11.766+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:10:11.769+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:10:11.768+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:10:11.781+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:10:11.779+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:10:11.782+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:10:11.809+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.049 seconds
[2025-01-12T13:10:42.066+0000] {processor.py:186} INFO - Started process (PID=261) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:10:42.068+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:10:42.071+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:10:42.071+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:10:42.089+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:10:42.084+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:10:42.091+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:10:42.109+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.049 seconds
[2025-01-12T13:11:12.634+0000] {processor.py:186} INFO - Started process (PID=326) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:11:12.635+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:11:12.638+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:11:12.637+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:11:12.654+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:11:12.650+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:11:12.657+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:11:12.686+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.058 seconds
[2025-01-12T13:11:43.441+0000] {processor.py:186} INFO - Started process (PID=391) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:11:43.443+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:11:43.445+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:11:43.445+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:11:43.460+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:11:43.456+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:11:43.461+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:11:43.477+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.041 seconds
[2025-01-12T13:12:14.197+0000] {processor.py:186} INFO - Started process (PID=456) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:12:14.198+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:12:14.201+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:12:14.200+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:12:14.215+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:12:14.211+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:12:14.216+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:12:14.232+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.040 seconds
[2025-01-12T13:12:45.196+0000] {processor.py:186} INFO - Started process (PID=527) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:12:45.198+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:12:45.202+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:12:45.201+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:12:45.223+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:12:45.219+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:12:45.225+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:12:45.256+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.065 seconds
[2025-01-12T13:13:15.716+0000] {processor.py:186} INFO - Started process (PID=592) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:13:15.717+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:13:15.719+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:13:15.719+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:13:15.734+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:13:15.731+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:13:15.735+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:13:15.751+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.040 seconds
[2025-01-12T13:13:46.055+0000] {processor.py:186} INFO - Started process (PID=657) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:13:46.056+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:13:46.058+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:13:46.058+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:13:46.087+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:13:46.083+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:13:46.089+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:13:46.106+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.056 seconds
[2025-01-12T13:14:16.793+0000] {processor.py:186} INFO - Started process (PID=724) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:14:16.794+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:14:16.797+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:14:16.796+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:14:16.811+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:14:16.808+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:14:16.812+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:14:16.838+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.050 seconds
[2025-01-12T13:14:46.887+0000] {processor.py:186} INFO - Started process (PID=789) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:14:46.888+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:14:46.889+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:14:46.889+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:14:46.902+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:14:46.899+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:14:46.904+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:14:46.918+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.036 seconds
[2025-01-12T13:15:17.678+0000] {processor.py:186} INFO - Started process (PID=854) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:15:17.679+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:15:17.680+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:15:17.680+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:15:17.693+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:15:17.690+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:15:17.695+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:15:17.722+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.049 seconds
[2025-01-12T13:15:48.346+0000] {processor.py:186} INFO - Started process (PID=919) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:15:48.347+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:15:48.348+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:15:48.347+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:15:48.360+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:15:48.357+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:15:48.361+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:15:48.376+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.035 seconds
[2025-01-12T13:16:19.109+0000] {processor.py:186} INFO - Started process (PID=984) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:16:19.110+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:16:19.111+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:16:19.110+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:16:19.126+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:16:19.123+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:16:19.127+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:16:19.157+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.053 seconds
[2025-01-12T13:16:49.919+0000] {processor.py:186} INFO - Started process (PID=1049) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:16:49.920+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:16:49.921+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:16:49.921+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:16:49.934+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:16:49.931+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:16:49.935+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:16:49.950+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.036 seconds
[2025-01-12T13:17:20.238+0000] {processor.py:186} INFO - Started process (PID=1118) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:17:20.239+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:17:20.240+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:17:20.240+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:17:20.255+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:17:20.251+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:17:20.256+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:17:20.281+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.048 seconds
[2025-01-12T13:20:44.834+0000] {processor.py:186} INFO - Started process (PID=74) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:20:44.839+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:20:44.844+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:20:44.843+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:20:44.877+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:20:44.871+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:20:44.880+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:20:44.914+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.092 seconds
[2025-01-12T13:21:15.652+0000] {processor.py:186} INFO - Started process (PID=139) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:21:15.653+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:21:15.656+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:21:15.655+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:21:15.672+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:21:15.667+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:21:15.673+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:21:15.690+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.044 seconds
[2025-01-12T13:21:46.192+0000] {processor.py:186} INFO - Started process (PID=204) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:21:46.192+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:21:46.194+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:21:46.193+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:21:46.211+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:21:46.207+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:21:46.214+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:21:46.234+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.047 seconds
[2025-01-12T13:22:16.628+0000] {processor.py:186} INFO - Started process (PID=269) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:22:16.630+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:22:16.631+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:22:16.631+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:22:16.660+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:22:16.654+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:22:16.662+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:22:16.687+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.069 seconds
[2025-01-12T13:22:46.945+0000] {processor.py:186} INFO - Started process (PID=334) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:22:46.946+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:22:46.947+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:22:46.946+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:22:46.963+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:22:46.959+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:22:46.964+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:22:46.993+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.053 seconds
[2025-01-12T13:23:17.520+0000] {processor.py:186} INFO - Started process (PID=399) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:23:17.521+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:23:17.522+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:23:17.522+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:23:17.537+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:23:17.534+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:23:17.539+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:23:17.569+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.055 seconds
[2025-01-12T13:23:47.733+0000] {processor.py:186} INFO - Started process (PID=464) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:23:47.734+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:23:47.735+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:23:47.735+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:23:47.750+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:23:47.745+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:23:47.751+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:23:47.776+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.049 seconds
[2025-01-12T13:24:18.654+0000] {processor.py:186} INFO - Started process (PID=529) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:24:18.655+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:24:18.657+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:24:18.657+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:24:18.671+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:24:18.666+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:24:18.673+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:24:18.700+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.051 seconds
[2025-01-12T13:24:48.776+0000] {processor.py:186} INFO - Started process (PID=594) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:24:48.777+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:24:48.779+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:24:48.779+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:24:48.794+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:24:48.789+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:24:48.796+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:24:48.824+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.053 seconds
[2025-01-12T13:25:18.861+0000] {processor.py:186} INFO - Started process (PID=658) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:18.861+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:25:18.863+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:25:18.862+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:18.876+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:25:18.874+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:25:18.877+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:18.893+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.037 seconds
[2025-01-12T13:25:20.251+0000] {processor.py:186} INFO - Started process (PID=664) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:20.252+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:25:20.253+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:25:20.253+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:20.276+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:25:20.273+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:25:20.278+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:20.305+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.060 seconds
[2025-01-12T13:25:22.554+0000] {processor.py:186} INFO - Started process (PID=682) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:22.555+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:25:22.556+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:25:22.556+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:22.581+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:25:22.575+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:25:22.583+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:22.605+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.055 seconds
[2025-01-12T13:25:53.356+0000] {processor.py:186} INFO - Started process (PID=744) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:53.357+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:25:53.358+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:25:53.358+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:53.376+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:25:53.372+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:25:53.378+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:25:53.405+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.054 seconds
[2025-01-12T13:26:23.641+0000] {processor.py:186} INFO - Started process (PID=809) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:26:23.642+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:26:23.643+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:26:23.642+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:26:23.656+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:26:23.653+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:26:23.657+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:26:23.685+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.049 seconds
[2025-01-12T13:26:53.717+0000] {processor.py:186} INFO - Started process (PID=873) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:26:53.718+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:26:53.719+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:26:53.719+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:26:53.731+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:26:53.728+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:26:53.732+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:26:53.757+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.045 seconds
[2025-01-12T13:27:03.076+0000] {processor.py:186} INFO - Started process (PID=879) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:27:03.077+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:27:03.079+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:27:03.078+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:27:03.099+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:27:03.096+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:27:03.100+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:27:03.127+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.056 seconds
[2025-01-12T13:27:33.218+0000] {processor.py:186} INFO - Started process (PID=944) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:27:33.220+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:27:33.222+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:27:33.221+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:27:33.240+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:27:33.236+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:27:33.242+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:27:33.259+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.046 seconds
[2025-01-12T13:29:34.733+0000] {processor.py:186} INFO - Started process (PID=70) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:29:34.734+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:29:34.737+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:29:34.737+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:29:34.764+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:29:34.757+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:29:34.766+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:29:34.794+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.069 seconds
[2025-01-12T13:30:05.638+0000] {processor.py:186} INFO - Started process (PID=135) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:30:05.640+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:30:05.642+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:30:05.642+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:30:05.714+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:30:05.666+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:30:05.738+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:30:05.794+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.165 seconds
[2025-01-12T13:30:36.316+0000] {processor.py:186} INFO - Started process (PID=200) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:30:36.317+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:30:36.319+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:30:36.319+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:30:36.333+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:30:36.330+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:30:36.334+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:30:36.360+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.050 seconds
[2025-01-12T13:31:06.446+0000] {processor.py:186} INFO - Started process (PID=265) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:31:06.447+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:31:06.449+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:31:06.449+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:31:06.465+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:31:06.462+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:31:06.466+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:31:06.494+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.053 seconds
[2025-01-12T13:31:36.588+0000] {processor.py:186} INFO - Started process (PID=330) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:31:36.588+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:31:36.591+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:31:36.591+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:31:36.605+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:31:36.602+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:31:36.606+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:31:36.633+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.050 seconds
[2025-01-12T13:32:06.772+0000] {processor.py:186} INFO - Started process (PID=395) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:32:06.773+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:32:06.776+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:32:06.775+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:32:06.789+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:32:06.786+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:32:06.790+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:32:06.805+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.038 seconds
[2025-01-12T13:32:36.997+0000] {processor.py:186} INFO - Started process (PID=459) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:32:36.998+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:32:37.001+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:32:37.000+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:32:37.018+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:32:37.013+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:32:37.019+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:32:37.035+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.042 seconds
[2025-01-12T13:32:43.395+0000] {processor.py:186} INFO - Started process (PID=483) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:32:43.396+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:32:43.399+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:32:43.399+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:32:43.434+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:32:43.428+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:32:43.436+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:32:43.457+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.066 seconds
[2025-01-12T13:34:22.925+0000] {processor.py:186} INFO - Started process (PID=73) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:34:22.927+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:34:22.930+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:34:22.930+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:34:22.952+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:34:22.947+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:34:22.955+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:34:22.996+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.080 seconds
[2025-01-12T13:34:54.030+0000] {processor.py:186} INFO - Started process (PID=137) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:34:54.032+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:34:54.036+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:34:54.035+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:34:54.073+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:34:54.060+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:34:54.076+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:34:54.098+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.075 seconds
[2025-01-12T13:35:24.597+0000] {processor.py:186} INFO - Started process (PID=202) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:35:24.598+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:35:24.601+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:35:24.601+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:35:24.616+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:35:24.612+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:35:24.618+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:35:24.647+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.058 seconds
[2025-01-12T13:35:55.146+0000] {processor.py:186} INFO - Started process (PID=267) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:35:55.147+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:35:55.149+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:35:55.148+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:35:55.164+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:35:55.161+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:35:55.165+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:35:55.194+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.054 seconds
[2025-01-12T13:36:25.921+0000] {processor.py:186} INFO - Started process (PID=332) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:36:25.923+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:36:25.927+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:36:25.927+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:36:25.946+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:36:25.943+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:36:25.948+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:36:25.977+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.062 seconds
[2025-01-12T13:36:56.410+0000] {processor.py:186} INFO - Started process (PID=397) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:36:56.411+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:36:56.414+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:36:56.414+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:36:56.427+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:36:56.423+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:36:56.428+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:36:56.455+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.051 seconds
[2025-01-12T13:37:26.697+0000] {processor.py:186} INFO - Started process (PID=463) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:37:26.698+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:37:26.701+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:37:26.700+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:37:26.714+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:37:26.711+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:37:26.716+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:37:26.744+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.052 seconds
[2025-01-12T13:37:57.739+0000] {processor.py:186} INFO - Started process (PID=528) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:37:57.740+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:37:57.742+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:37:57.742+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:37:57.754+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:37:57.750+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:37:57.755+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:37:57.770+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.037 seconds
[2025-01-12T13:38:28.224+0000] {processor.py:186} INFO - Started process (PID=593) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:38:28.225+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:38:28.227+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:38:28.227+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:38:28.240+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:38:28.237+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from Capstone_project import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'Capstone_project'
[2025-01-12T13:38:28.242+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:38:28.267+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.048 seconds
